include/wx/osx/textctrl.h

```diff
@@ -166,6 +167,8 @@ class WXDLLIMPEXP_CORE wxTextCtrl: public wxTextCtrlBase
 private :
     wxMenu  *m_privateContextMenu;
 
+    // stores if the text control replaces new lines with spaces or not
+    bool m_replaceNewLine;
```

Comment:
I'd slightly prefer to avoid storing this here in addition to storing it inside the actual implementation. Could we add a function to the "text peer" to return this value as needed? Something like `NeedsEnterKey()`, perhaps?

Reply:
sound good, will implement that

Reply:
and commited!